libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/extensions/coroutines/MapboxNavigation.kt

```diff
@@ -0,0 +1,48 @@
+package com.mapbox.navigation.dropin.extensions.coroutines
```

Comment:
nit: Rename `MapboxNavigation.kt` to something unique

`MapboxCoroutineExtensions.kt`, and delete the `coroutines` package?

Reply:
What's wrong with packages? Why unique filename? We never import extension methods by filename.

Reply:
Better IDE integration? What IDE are you using? 
In Android Studio you can select "Classes" tab in the search dialog.

![Screen Shot 2022-03-02 at 3 24 56 PM](https://user-images.githubusercontent.com/2678039/156443748-f5556a7c-3247-40ee-8031-edf6ed0ad13b.png)

As for the Files... 

![Screen Shot 2022-03-02 at 3 23 18 PM](https://user-images.githubusercontent.com/2678039/156443586-fcf16290-cea6-48ed-9a95-4459afd58534.png)

...yeah... you are right... this is super confusing. How will we ever figure out which file is which?
<img width="250" src="https://c.tenor.com/o5CbgYywhaoAAAAd/justin-timberlake-stare.gif" />

Reply:
We are only one extension function or data class away from the `core.MapboxNavigation` becoming a file and not a class which would not allow us to filter it out this way anymore.

What's the problem with adding a prefix/suffix with the desired purpose of the new file? I don't understand why we should go through this cognitive hoops and compare packages when we search.